eventsource/resources/message2.py

```diff
@@ -4,7 +4,6 @@ def main(request, response):
     response.headers.set(b'Content-Type', b'text/event-stream')
     response.headers.set(b'Cache-Control', b'no-cache')
 
-    response.explicit_flush = True
```

Comment:
So it sems lke these tests using `explicit_flush` might be broken? Should we replace this with a mechanism that works in the way they expect (or otherwise edit the tests to call write once)?

Reply:
Yes and no.

They have always been broken in the sense that `flush` and `explicit_flush` do not do what they expect. But in practice, they seem to only require content to be flushed out before `sleep`, and unbuffered write satisfies that, which also explains why no test result changes.

Reply:
I'm merging this for now. Feel free to open a new issue on this if you think it's important.